loomi/regulariza-morador-client
# Feature/form layout about you

## Resident Form

PS.: The success Modal on finish will be implemented with Dashboard task

---

**What I did:**

- Form
  - Integrated with json server
  - Save answers on Local Storage

**How to test:**

1. Pull this branch locally
2. Start the client with `yarn start` and the mock server with `yarn mock-server`
3. Begin to answer the questions and see if what you type are getting saved on local storage
4. Try to close the tab on the middle of one question to see if, when you come back, the input will be the same as what you left
5. The answers are submitted on "Avançar", see if they are appearing on db.json file.
6. Test other things you consider important
